libs/newsletters-data-client/src/lib/draft-storage/InMemoryDraftStorage.ts

```diff
@@ -33,6 +33,10 @@ export class InMemoryDraftStorage implements DraftStorage {
 			listId: this.getNextId(),
 			creationTimeStamp: Date.now(),
 			meta: undefined,
+			brazeCampaignCreationsStatus: 'NOT_REQUESTED',
+			ophanCampaignCreationsStatus: 'NOT_REQUESTED',
+			tagCreationsStatus: 'NOT_REQUESTED',
+			signupPageCreationsStatus: 'NOT_REQUESTED'
```

Comment:
The values don't seem to get populated when running locally 🤔 . Not sure why. But It would probably be better to use the `defaultNewsletterValues` in libs/newsletters-data-client/src/lib/draft-to-newsletter.ts - that way you won't need to change both the DraftStorage implementations.

Reply:
Right - I like writing less code but I guess I will need to update the schema to exclude these properties for the draft, otherwise it would always fail validation in the draft state.

I will take a look

Reply:
I have moved the default values logic into the workflow `executeCreate` function to reduce duplication. I think it's simpler to set these at draft creation time and this gives us the option of marking / inferring completion of some async workflow tasks at data collection time. 

Regarding the failure to populate, I cannot recreate. Both before and after the change, creating a draft (after setting a  name), results in the values being set. I have tested this in both inMem & s3. Happy to jump on  a call to see if we can troubleshoot together?

Reply:
I was using the existing hard-coded test drafts and running through the launch step to see if the values were set on the launched newsletters - but I see now why that wouldn't work - you are setting the values at draft creation, and the test drafts don't have them set, so they aren't set on the newsletters resulting from the launch.

related point - I noticed I left a bug in the `withDefaultNewsletterValuesAndDerivedFields` in "draft-to-newsletter" - the value for "status" is only being set if the draft has a value for `renderingOptions` - if not the draft fails the 'is draft ready' check

Reply:
Ahh right - I will update the hard-coded ones
Good spot